### PR TITLE
Change visibility of PolygonSpriteBatch.switchTexture() to protected

### PR DESCRIPTION
**Pull request by TCreutzenberg**
Change visibility of PolygonSpriteBatch.switchTexture() to protected

Use case: On texture switch pass the new texture size to a shader

(SpriteBatch.switchTexture is already protected)

### CODE REVIEW

File: gdx/src/com/badlogic/gdx/graphics/g2d/PolygonSpriteBatch.java [hunk 1]
```diff
@@ -1310,7 +1310,7 @@ protected void setupMatrices () {
 		}
 	}
 
-	private void switchTexture (Texture texture) {
+	protected void switchTexture (Texture texture) {
```
Comment (tommyettinger) [inline gdx/src/com/badlogic/gdx/graphics/g2d/PolygonSpriteBatch.java:1313]: If this method is already protected in SpriteBatch, it makes sense to have the APIs be as similar as possible. Approved!

**Review by tommyettinger — APPROVED** [review]
Looks small and solid, and seems to pass Spotless without issue. Thanks for updating the changelog, too!

**Comment by crykn** [comment]
@TCreutzenberg could you move the CHANGELOG entry to 1.13.1 please, then we can go ahead and merge this.

**Comment by TCreutzenberg** [comment]
@crykn Done.

**Pull request merged**